ui/components/Header.js

```diff
@@ -211,20 +210,23 @@ function K8sContextMenu({
         </div>
       </IconButton>
 
-      <Slide direction="down" timeout={300} in={open} style={{ position : "absolute", left : "-5rem", zIndex : "-1", bottom : "-75%", transform : showFullContextMenu ? `translateY(${transformProperty}%)`: "translateY(0)" }} mountOnEnter unmountOnExit>
+      <Slide direction="down"  timeout={400} in={open} style={{ position : "absolute", left : "-5rem", zIndex : "-1", bottom : "-100%", transform : showFullContextMenu ? `translateY(${transformProperty}%)`: "translateY(0)" }} mountOnEnter unmountOnExit>
```

Comment:
If the switcher looks detached form the title bar then changing margin here in styles may help.

Reply:
Also @Abhishek-kumar09 instead of Inline CSS styles, would love to know your opinion on using a separate CSS file? 👀

Reply:
@adithyaakrishna I guess, Our showfullcontextMenu state then has to pass in themes which I think is not desirable.

Reply:
Yeah we do try to avoid the inline CSS as much as possible, this code is some old addition. 
We can use `makeStyles` here.